### DISCUSSION TITLE
StyleX & Autoprefixer

### DISCUSSION THREAD

**Discussion by ghost** [facebook/stylex#223] [Q&A] [answered]
Does StyleX include autoprefixing for the @keyframes rule and other CSS properties?

**Comment by nmn** [comment, marked as answer]
You have to set up autoprefixr separately.

**Reply by steida** [reply]
How? I tried to copy past Next.js Tailwind example to enable postcss with autoprefixer, but it still does not prefix. Any idea, please?

**Reply by nmn** [reply]
I'm working on this on my end so will report back when it's done. It might need an update to the Next.js plugin.

**Reply by nmn** [reply]
After some digging, it looks like the NextJS postcss runs *before* the stylex styles are inserted into the generated stylesheet.

This will take a bit longer to address, but I'm working on it.

**Reply by necolas** [reply]
Why don't we just prefix in stylex? Would save us and users a lot of trouble, instead users of every bundler having to do this themselves. We need to add polyfills for certain style props anyway
